hwdb.d/60-autosuspend.hwdb

```diff
@@ -87,6 +87,10 @@ usb:v2CB7p0007*
  ID_AUTOSUSPEND=1
  ID_AUTOSUSPEND_DELAY_MS=7000
 
+# Intel Corp. Bluetooth wireless interface
```

Comment:
Not sure if the change is acceptable or not, but
- please create an appropriate section, e.g. Buletooth, 
- and add brief comment in the .hwdb file,
- please use `ID_AUTOSUSPEND=0`, instead of using spurious negative value for `_DELAY_MS`, and update rules.d/60-autosuspend.rules.

Reply:
OK, could you also help me to find out what's wrong with the test?
I don't see that `/home/runner/work/systemd/systemd/build/meson-logs/testlog.txt` anywhere.
```
Summary of Failures:

1059/1548 systemd:dist / parse-hwdb                                                              FAIL             3.00s   exit status 1

Ok:                 1541
Expected Fail:      0   
Fail:               1   
Unexpected Pass:    0   
Skipped:            6   
Timeout:            0   

Full log written to /home/runner/work/systemd/systemd/build/meson-logs/testlog.txt
Error: Process completed with exit code 1.

Reply:
Not sure, but the test does not expect negative values for the property.

Reply:
I thought better way to fix might be prevent autosuspend for the adapter in the first place:
https://github.com/systemd/systemd/pull/27771